doc/manuals/user/filtering.md

```diff
@@ -188,7 +188,26 @@ described in detail in [the following section](geolocation.md#geo-located-querie
 
 [Top](#top)
 
-### String filter
+### String filters
```

Comment:
As a consecuence of changing the tittle references to `#string-filter` has to change to `#string-filters`. I have located at least two ones:

```
fermin@neodeb:~/src/fiware-orion/doc/manuals$ grep string-filter -R *
user/v1_v2_coexistence.md:[the following section](filtering.md#string-filter) for details.
user/filtering.md:  * [String query filter](#string-filter)
```